core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -515,4 +519,14 @@ class EMConfig {
     @Experimental
     @Cfg("Enable EvoMaster to generate, use, and attach complete objects to REST calls, rather than just the needed fields/values")
     var enableCompleteObjects = false
+
+    @Experimental
+    @Cfg("Whether to track individual")
+    var enableTrackIndividual = false
+
+    @Experimental
+    @Cfg("Whether to track evaluated individual. " +
+            "Note that we recommend that set enableTrackIndividual false when enableTrackEvaluatedIndividual is true since information of individual is part of evalauted individual")
```

Comment:
"recommend"? or enforced? don't you throw an exception if both are true?

Reply:
modified.